src/applications/vaos/tests/new-appointment/redux/helpers/formSubmitTransformers.unit.spec.js

```diff
@@ -17,7 +17,7 @@ describe('VAOS data transformation', () => {
             morning: true,
           },
           email: 'test@va.gov',
-          visitType: 'office',
+          visitType: 'clinic',
```

Comment:
@vbahinwillit I found some broken tests when I ran the full suite, we needed to update some of them due to the office -> clinic change.